x-pack/plugins/maps/public/connected_components/timeslider/time_utils.ts

```diff
@@ -46,7 +46,7 @@ function getScaledDateFormat(interval: number): string {
 }
 
 export function epochToKbnDateFormat(epoch: number): string {
-  const dateFormat = getUiSettings().get('dateFormat', 'MMM D, YYYY @ HH:mm:ss.SSS');
+  const dateFormat = getUiSettings().get('dateFormat', 'MMM D, YYYY @ HH:mm:ss');
```

Comment:
I don't think we want to change this formatting. For small time ranges, showing milliseconds is important.